src/visualization/types.jl

```diff
@@ -348,58 +348,77 @@ end
 
 # specializes the PlotData2D constructor to return an PlotData2DTriangulated for any type of mesh.
 function PlotData2DTriangulated(u, mesh, equations, dg::DGSEM, cache;
-                                solution_variables=nothing, nvisnodes=2*polydeg(dg))
+                                solution_variables=nothing, nvisnodes=nothing,
+                                elevations=nothing, slice=:yz, point=[0,0,0])
```

Comment:
```suggestion
                                elevations=nothing, slice=:yz, point=(0,0,0))
```
To avoid allocations of a small array?